cdc/processor/pipeline/sink.go

```diff
@@ -373,3 +398,30 @@ func (n *sinkNode) Destroy(ctx pipeline.NodeContext) error {
 	n.flowController.Abort()
 	return n.sink.Close(ctx)
 }
+
+// retryEmitRow2Sink is a blocking method, it retries emit rows to sink until success
+// We must call this method before we call flushSink
+func (n *sinkNode) retryEmitRow2Sink(ctx context.Context) error {
+	start := time.Now()
+	lastTimeToRetry := start
+	err := retry.Do(ctx, func() error {
+		// We must retry until success when emitRow2Sink return false.
+		ok, err := n.emitRow2Sink(ctx)
+		if err == nil && !ok {
+			err = cerror.ErrSinkBlocking.FastGenByArgs()
+		}
+		cost := time.Since(start)
+		retryInterval := time.Since(lastTimeToRetry)
+		if cost >= sink.SinkBlockingWarnLogDuration && retryInterval >= sink.SinkBlockingWarnLogInterval {
+			lastTimeToRetry = time.Now()
+			log.Warn("sink blocking too long", zap.Duration("blocking time(s)", cost))
```

Comment:
```suggestion
			log.Warn("sink blocking too long", zap.Duration("duration", cost))
```

Use duration for slow logs.